lib/pbio/drv/usb/usb_stm32.c

```diff
@@ -121,14 +136,135 @@ void pbdrv_usb_stm32_handle_vbus_irq(bool active) {
     process_poll(&pbdrv_usb_process);
 }
 
+/**
+ * Queues data to be transmitted via USB.
+ * @param data  [in]        The data to be sent.
+ * @param size  [in, out]   The size of @p data in bytes. After return, @p size
+ *                          contains the number of bytes actually written.
+ * @return                  ::PBIO_SUCCESS if @p data was queued, ::PBIO_ERROR_AGAIN
+ *                          if @p data could not be queued at this time (e.g. buffer
+ *                          is full), ::PBIO_ERROR_INVALID_OP if there is not an
+ *                          active USB connection or ::PBIO_ERROR_NOT_SUPPORTED
+ *                          if this platform does not support USB.
+ */
+pbio_error_t pbdrv_usb_stm32_stdout_tx(const uint8_t *data, uint32_t *size) {
+    uint8_t *ptr = usb_stdout_buf;
+    uint32_t ptr_len = sizeof(usb_stdout_buf);
+
+    // TODO: return PBIO_ERROR_INVALID_OP if not connected
+
+    if (usb_stdout_sz) {
+        return PBIO_ERROR_AGAIN;
+    }
+
+    pbio_uuid128_le_copy(ptr, pbio_pybricks_command_event_char_uuid);
```

Comment:
Other than the cases where we have to use RFC 4122 format to be compatible with Windows, I would rather use UUIDs big-endian.